cli.go

```diff
@@ -299,6 +311,9 @@ Configuration:
 
   callbackport=<string>
       Optional port to to use in OIDC redirect_uri (default: the value set for port).
+
+  no-launch
```

Comment:
nit: I wonder if using `open-browser` (default of true) for the name would make the behavior of this option more clear? I see that our docs mention both opening and launching, so I'm okay with either. Curious to hear others' thoughts.

It would look like:
```
vault login -method=oidc -path=oidc role=test open-browser=false
```

Reply:
Since this is a new parameter, using the zero value of `false` as the default value might be a bit more intuitive. How about `skip_browser` with default of `false`?